packages/mesos/extra/patches/0007-Disabled-hostname-validation-for-master-scheduler-communication.patch

```diff
@@ -0,0 +1,129 @@
+From 89c1ac9d1d20d6b258c469d38040f255b54c3768 Mon Sep 17 00:00:00 2001
+From: Till Toenshoff <toenshoff@me.com>
+Date: Fri, 2 Aug 2019 23:27:35 +0200
+Subject: [PATCH] Disabled hostname validation for master to scheduler
+ communication.
+
+Some schedulers may run in non-host networking mode containers, in which
+case the default-provisioned certificate from the DC/OS CA appears
+incorrect.
```

Comment:
> in which case the default-provisioned certificate from the DC/OS CA appears incorrect.

Let's maybe try to be more specific? Maybe "in which case the default-provisioned certificate from the DC/OS CA does not contain a corresponding subject alternative name, making hostname validation fail"?